pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -356,8 +356,6 @@ protected void validateTopicName(String property, String namespace, String encod
                     topic, e);
             throw new RestException(Status.PRECONDITION_FAILED, "Topic name is not valid");
         }
-
-        this.topicName = TopicName.get(domain(), namespaceName, topic);
```

Comment:
Unrelated changes?

Reply:
Yes, it's a repeated code (see line 353). I've modified this method before, when I changed it back, I kept this line removed.